### DISCUSSION TITLE
ArangoDB Database Setup on Vercel

### DISCUSSION THREAD

**Discussion by koji98** [vercel/next.js#34390] [Help]
Hello, I am wondering if it's possible to setup a database on Vercel, specifically ArangoDB. My website currently depends on it and lives on a DigitalOcean Droplet since I can configure the environment. I would love to deploy the new version to Vercel to take advantage of so many of your awesome features, but I need to setup ArangoDB to do so. I see ArangoDB is currently not a part of the [database solutions](https://vercel.com/docs/concepts/solutions/databases) provided. I understand I could setup ArangoDB Oasis to access the database remotely, but it's expensive and so is not an option for me. Because of that i'm currently using the community edition which I install manually on the server. Is there anyway to do something similar on Vercel?

**Comment by fandyajpo** [comment]
koji, is it save if i public  the server that has arangodb and my nextjs app will add server-ip to the environment to connect the arango, have u try it?

**Reply by koji98** [reply]
I ended up keeping my setup on Digital Ocean instead of migrating to Vercel. But you definitely can host an ArangoDB instance on a server and connect to that directly

**Reply by fandyajpo** [reply]
thanks koji :) 👌🏻
